src/server.py

```diff
@@ -4,10 +4,14 @@
 import socket
 from client import PORT
 from message import Request, Response
+import os
 
 HTML_PROTOCOL = 'HTTP/1.1'
 CRLF = '\r\n'
 
+# PWD = os.dir_name(__file__)
```

Comment:
This is a totally valid way to get hold of the current working directory.  You could then define the path to your home directory relative to this location using `os.path.join`.